src/main/java/com/arcbees/website/client/application/products/chosen/ChosenView.ui.xml

```diff
@@ -306,17 +316,17 @@
             </h3>
             <ul class="{page.style.social}">
                 <li>
-                    <a href="#" class="{fonts.icons.iconFb}">
+                    <a href="#" title="facebook" class="{fonts.icons.iconFb}">
```

Comment:
Keep it like that, but when the title is the same as the text inside the link, it's not required, it's even more annoying than anything : http://www.searchenginejournal.com/how-to-use-link-title-attribute-correctly/